src/altinn-app-frontend/src/shared/resources/dataLists/index.d.ts

```diff
@@ -47,3 +47,46 @@ export interface ISetSort {
   sortColumn: string;
   sortDirection: SortDirection;
 }
+
+export interface IDataList {
+  listItems: any;
+  _metaData: IDataListPaginationData;
+}
+
+export interface IDataLists {
+  [key: string]: IDataListData;
+}
+
+export interface IDataListActualData {
+  listItems: any;
```

Comment:
I think a bit more specific type could be `Record<string, string>[]`, i.e. a list of objects with string keys and string values.